### PR TITLE
Add tsparticles via single package.json w/ npm auto-update

### PR DESCRIPTION
**Pull request by matteobruni**
# Pull request template for new library

Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required):
* Official website (optional, not the repository):
* NPM package url (optional):
* License and its reference:
* GitHub / Bitbucket popularity (required):
  * Count of watchers:
  * Count of stars:
  * Count of forks:
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week:
  * Downloads in the last month:

## Essential checklist

* [x] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [ ] The lib has notable popularity
  * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [ ] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [x] Close corresponding issue in commit message
* [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/tsparticles/package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  "author": "Matteo Bruni",
+  "license": "MIT",
+  "files": [
+    "dist/**/*"
+  ],
+  "bugs": {
+    "url": "https://github.com/matteobruni/tsparticles/issues"
+  },
+  "main": "dist/tsparticles.min.js",
+  "devDependencies": {
```
Comment (xtuc) [inline ajax/libs/tsparticles/package.json]: remove devDependencies and dependencies

File: ajax/libs/tsparticles/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  ],
+  "npmName": "tsparticles",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ]
```
Comment (xtuc) [inline ajax/libs/tsparticles/package.json]:
```suggestion
  ],
```

File: ajax/libs/tsparticles/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "author": "Matteo Bruni",
+  "license": "MIT",
+  "main": "dist/tsparticles.min.js"
```
Comment (MattIPv4) [inline ajax/libs/tsparticles/package.json]:
```suggestion
  "filename": "dist/tsparticles.min.js"
```
We use the `filename` property for the default file, not `main`.

File: ajax/libs/tsparticles/package.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "tsparticles",
+  "version": "1.6.0",
```
Comment (MattIPv4) [inline ajax/libs/tsparticles/package.json]:
```suggestion
```

Please do not include a version attr by default, the bot will populate this automatically once it starts pulling in versioned files for the library.

File: ajax/libs/tsparticles/package.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "author": "Matteo Bruni",
+  "license": "MIT",
+  "filename": "dist/tsparticles.min.js"
```
Comment (MattIPv4) [inline ajax/libs/tsparticles/package.json]:
```suggestion
  "filename": "tsparticles.min.js"
```

As `basePath` is used in the auto-update config, this file will be at the root.

**Pull request merged**